frontend/packages/schema-editor/src/components/SchemaTree/SchemaNode/ActionButtons/ActionButtons.tsx

```diff
@@ -32,12 +33,22 @@ export const ActionButtons = ({ pointer, className }: ActionButtonsProps) => {
           onClick={convertToReference}
         />
       )}
-      <ActionButton
-        color='danger'
-        icon={<TrashIcon />}
-        titleKey='general.delete'
-        onClick={deleteNode}
-      />
+      {isDefinition(node) ? (
```

Comment:
`isDefinition` er vel ikke nok til å utlede om definisjonen er i bruk? `SchemaModel`-klassen har en metode som heter `hasReferringNodes` som vi kan prøve å bruke her. Hvis du gjør den om til å være `public`, blir den tilgjengelig på `savableModel`-variabelen. Legg gjerne til en test for dette også.